.dev/docker/fmt/Dockerfile

```diff
@@ -1,12 +0,0 @@
-# Azure needs node shadow, sudo and the label
```

Comment:
I would prefer to keep this file, so we can switch back easily if we can't/don't want to use the Azure VM image any more

Reply:
I had also shortly thought about it, but then decided against it, because the Azure images seem to already contain Clang-Format in the older versions, i.e. it is foreseeable that in the upcoming LTS in just less than 2 years will also contain the Clang-Format. In addition, it might be irritating that we have a format Docker image but don't use it and that's strange anyway because it's based on `node:alpine` and not on e.g. `alpine:latest` (do we need Node.js for anything?) and you don't see the installation of Clang-Format there explicitly either.

Reply:
`node:lts-alpine` is used because of Azure (https://github.com/PointCloudLibrary/pcl/pull/3982)
Ok, I can transfer this Dockerfile "recipe" to dockerhub (https://hub.docker.com/r/pointcloudlibrary/fmt) and add a note that we are currently not using it